source/testing_govuk_pay/index.html.md.erb

```diff
@@ -80,15 +85,17 @@ Mock card numbers do not work with your live account.
 |Invalid CVC code|4000000000000127|Visa| Credit or debit |
 |General error|4000000000000119|Visa| Credit or debit |
 
-##### Testing recurring payments
+##### Testing recurring payments on a test account
```

Comment:
Looks like this change is breaking a link somewhere

Reply:
@kbottla This should be fixed now